### PR TITLE
Add Klarna method to RTP

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+type: object
+title: Session Token
```
Comment (ashkarpetin) [inline openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml:2]:
```suggestion
title: Klarna
```

**Review by ashkarpetin — DISMISSED** [review]

**Review by ninjasimon — DISMISSED** [review]

File: openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    type: string
+    description: The feature name.
+    enum:
+      - Klarna
+  clientToken:
+    type: string
+    description: Klarna client token.
+  sessionId:
+    type: string
+    description: Klarna session token.
```
Comment (colinbird) [inline openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml:18]: session id or token?

**Review by colinbird — DISMISSED** [review]

File: openapi/components/schemas/ReadyToPay/Methods/ReadyToPayKlarnaMethod.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  method:
+    type: string
+    description: The payment method.
+    enum:
+      - Klarna
+  feature:
+    type: object
+    nullable: true
+    description: |
+      The specific feature (for example, digital wallet or a processor) of this method. If method doesn't have any features it will be null.
```
Comment (noon-dawg) [inline openapi/components/schemas/ReadyToPay/Methods/ReadyToPayKlarnaMethod.yaml:15]: nitpick: Avoid using contractions ("isn't", "don't", "shouldn’t" ... ) and parenthesis, where possible

> Parenthesis "( )": Avoid using parenthesis in text when possible. Readers interpret this in different ways and it disrupts the flow of a sentence or paragraph. 

```suggestion
      The specific feature of this method. For example, a digital wallet or a processor. If the method does not have any features, this value will be null.
```

File: openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml (hunk 2 above)
Comment (dara123) [reply openapi/components/schemas/ReadyToPay/Features/KlarnaFeature.yaml:18]: should be id

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**